torch/csrc/jit/mobile/nnc/aot_compiler.cpp

```diff
@@ -82,6 +83,17 @@ std::unique_ptr<Function> compileMethod(
     output.sizes_ = getConstSizes(ba.buf());
     // TODO: assert the output is a buffer and not a scalar
     output.dtype_ = ba.buf()->dtype().scalar_type();
+    if (isQIntType(output.dtype_)) {
+      // Supporting only static qscale/qzero
+      output.qscale_ =
+          to<DoubleImm>(torch::jit::tensorexpr::IRSimplifier::simplify(
+                            ba.buf()->qscale()))
```

Comment:
Should we simplify it whenever we create the buf, would it be cleaner (if it's possible at all)?